rules/cloud/okta/okta_admin_role_assignment_created.yml

```
@@ -0,0 +1,21 @@
+title: Okta Admin Role Assignment Created
+id: 139bdd4b-9cd7-49ba-a2f4-744d0a8f5d8c
+status: test
```

For new rules we start with the `experimental` status.
```suggestion
status: experimental
```